src/sentry/static/sentry/app/utils/theme.jsx

```diff
@@ -91,14 +91,14 @@ const theme = {
     sidebar: 1010,
     orgAndUserMenu: 1011,
 
-    // tooltips and hovercards
-    tooltip: 1070,
-
     // Sentry user feedback modal
     sentryErrorEmbed: 1090,
 
     modal: 10000,
     toast: 10001,
+
+    // tooltips and hovercards can be inside modals sometimes.
+    tooltip: 10002,
```

Comment:
How does this affect tooltips app-wide?

Reply:
I think this is one of the first situations where we need to show tooltips inside a modal window. Keeping focus/hover on an element when a modal is open is pretty hard so I don't forsee this causing many problems.